src/constants.ts

```diff
@@ -1,25 +1,41 @@
 // Chain names and ID's
-export const CHAIN_IDs = {
+export const TESTNET_SEPOLIA_CHAIN_IDs = {
+  SEPOLIA: 11155111,
+  OPTIMISM_SEPOLIA: 11155420,
+  ZK_SYNC_SEPOLIA: 300,
+  BASE_SEPOLIA: 84532,
+  ARBITRUM_SEPOLIA: 421614,
+  POLYGON_AMOY: 80002,
```

Comment:
```suggestion
  POLYGON_AMOY: 80002,
  BASE_SEPOLIA: 84532,
  ARBITRUM_SEPOLIA: 421614,
```

Reply:
though tbf we have SEPOLIA and OP_SEPOLIA up the top here so there's no numeric ordering anyway :\